README

```diff
@@ -65,11 +65,25 @@ Usage sof-logger <option(s)> <file(s)>
 -p			Get traces from stdin, instead of the default 
 			"/sys/kernel/debug/sof/etrace"
 -c			Set timestamp clock in MHz
+-e			Enable checking firmware version with default verification file 
+			"/sys/kernel/debug/sof/fw_version"
+-v ver_file		Enable checking firmware version with ver_file file, 
```

Comment:
As discussed in https://github.com/thesofproject/soft/issues/111 we decided to introduce this feature in 2 stages, to allow for parallel implementation from FW and kernel sides and to ensure that behaviour doesn't break anything in case of misalignment.
The version file is there to allow custom path to file dumped by driver, as our diagnostic driver on Windows uses different path than kernel one.